mne/surface.py

```diff
@@ -1736,27 +1736,11 @@ def _marching_cubes(image, level, smooth=0):
     return out
 
 
-def _get_surface_RAS_volumes(base_image, subject_from, subjects_dir):
-    """Check if the image is in Freesurfer surface RAS space."""
-    import nibabel as nib
-    if not isinstance(base_image, nib.spatialimages.SpatialImage):
-        base_image = nib.load(base_image)
-    base_image = nib.MGHImage(
-        _get_img_fdata(base_image).astype(np.float32), base_image.affine)
-    fs_t1 = nib.load(op.join(subjects_dir, subject_from, 'mri', 'T1.mgz'))
-    if not np.allclose(base_image.affine, fs_t1.affine, atol=1e-6):
-        raise RuntimeError('The `base_image` is not aligned to Freesurfer '
-                           'surface RAS space. This space is required as '
-                           'it is the space where the anatomical '
-                           'segmentation and reconstructed surfaces are')
-    return base_image, fs_t1
-
-
 def _warn_missing_chs(montage, dig_image, after_warp):
     """Warn that channels are missing."""
     # ensure that each electrode contact was marked in at least one voxel
     missing = set(np.arange(1, len(montage.ch_names) + 1)).difference(
-        set(np.unique(_get_img_fdata(dig_image))))
+        set(np.unique(np.array(dig_image.dataobj))))
```

Comment:
`np.asanyarray` is preferred as it won't necessarily make a copy (unless you are going to modify values, in which case `np.array` is fine -- but here you're not), and it's also nibabel's recommendation:

https://nipy.org/nibabel/reference/nibabel.dataobj_images.html#nibabel.dataobj_images.DataobjImage.get_data